src/Libraries/CoreNodes/Math.cs

```diff
@@ -58,6 +72,29 @@ public static IList RandomList(int amount)
             return result;
         }
 
+        /// <summary>
+        ///     Produces a list containing the given amount of random doubles
+        ///     in the deffined range of [0, 1) based on an intial seed value.
+        /// </summary>
+        /// <param name="amount">Amount of random numbers the result list will contain.</param>
+        /// <param name="value1">One end of the range for the random number.</param>
+        /// <param name="value2">One end of the range for the random number.</param>
+        /// <param name="seed">Seed value for the random number generator.</param>
+        /// <returns name="number">List of random numbers in the range.</returns>
+        /// <search>random,listcontains,seed</search>
+        public static IList RandomList(int amount, [DefaultArgument("0.0")] double value1, [DefaultArgument("1.0")] double value2, [DefaultArgument("1")] int seed)
```

Comment:
What attribute is this?
Is it the designscript one?
If it is our own attribute then I think you can disregard my comment about the overload/default paramter issue.

Reply:
I do recall a concerted effort at 2.x timeframe to remove overloads from the library - maybe @aparajit-pratap can say more there.